tests/test_client.py

```diff
@@ -902,6 +902,15 @@ def test_run_detector():
     assert len(responses.calls) == 2
 
 
+@responses.activate
+def test_run_detector_secondary_raster():
+    add_mock_detector_run_responses(1)
+    add_mock_operations_responses("success")
+    client = _client()
+    client.run_detector(1, 2, 3)
+    assert len(responses.calls) == 2
```

Comment:
I know it's the same in the test above, but shouldn't this be 3? Because I see 3 mocks in `add_mock_detector_run_responses`; also, while we are at it, I'm wondering

Reply:
Ah good catch. That was due to `add_mock_detector_run_responses` redefining `op_id=43`:
- This meant that the only operation status check performed was the one mocked by `add_mock_operations_responses`, which returns `success`. So we had only 2 calls (detector run and status check).

I've removed the `op_id` override in favor of `OPERATION_ID`, adapted the call count and removed the un-needed `add_mock_operations_responses` from both tests.

Reply:
did you push? I cannot see changes GH